packages/moonstone/IncrementSlider/IncrementSlider.js

```diff
@@ -96,7 +95,7 @@ const IncrementSliderBase = kind({
 		'data-webos-voice-group-label': PropTypes.string,
 
 		/**
-		 * When `true`, the knob displays selected and can be moved using 5-way controls.
+		 * Displays the knob in its selected state, and it can be moved using 5-way controls.
```

Comment:
How about: 'Sets knob to selected state and allows it to move via 5-way controls'?